sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/aio/_schema_registry_client_async.py

```diff
@@ -136,19 +138,36 @@ async def register_schema(
             content_type=kwargs.pop(
                 "content_type", "application/json; serialization={}".format(format)
             ),
-            **http_request_kwargs
+            **http_request_kwargs,
         )
 
         response = await self._generated_client.send_request(request, **kwargs)
         response.raise_for_status()
         return _parse_response_schema_properties(response, format)
 
+    @overload
     async def get_schema(self, schema_id: str, **kwargs: Any) -> Schema:
-        """
-        Gets a registered schema by its unique ID.
-        Azure Schema Registry guarantees that ID is unique within a namespace.
+        ...
+
+    @overload
+    async def get_schema(
+        self, *, group_name: str, name: str, version: int, **kwargs
+    ) -> Schema:
+        ...
+
+    @distributed_trace_async
+    async def get_schema(self, *args: Union[str, int], **kwargs: Any) -> Schema:
+        """Gets a registered schema. There are two ways to call this method:
+        1) To get a registered schema by its unique ID, pass the `schema_id` parameter and any optional
```

Comment:
What optional keyword args are there when passing in schema ID?

Reply:
so optional keywords are ones that are reserved by azure-core that users can [pass in for the request itself](https://github.com/Azure/azure-sdk-for-python/blob/b37efe5deb6aaa096f039a565e7222201257f0f8/sdk/core/azure-core/azure/core/rest/_rest_py3.py#L68). for ex, params/json/content/data.

Yep, both overloads would appear in the docs!